api/api-webhook-v1/src/test/groovy/com/thoughtworks/go/apiv1/webhook/request/push/BitBucketCloudPushRequestTest.java

```diff
@@ -93,6 +96,26 @@ void shouldGuessTheWebhookUrls(String body) {
                 );
     }
 
+    @ParameterizedTest
+    @FileSource(files = "/bitbucket-payload.json")
+    void shouldReturnScmNameIfAny(String body) {
+        Request request = newRequestWithAuthorizationHeader("repo:push", "", body);
+
+        BitBucketCloudPushRequest bitBucketCloudPushRequest = new BitBucketCloudPushRequest(request);
+        QueryParamsMap map = mock(QueryParamsMap.class);
+        when(request.queryMap()).thenReturn(map);
+        when(map.hasKey(KEY_SCM_NAME)).thenReturn(false);
+        assertThat(bitBucketCloudPushRequest.getScmNames()).isEmpty();
+
+        when(request.queryMap()).thenReturn(map);
+        when(map.hasKey(KEY_SCM_NAME)).thenReturn(true);
+
+        QueryParamsMap value = mock(QueryParamsMap.class);
+        when(map.get(KEY_SCM_NAME)).thenReturn(value);
+        when(value.values()).thenReturn(singletonList("scm1").toArray(new String[]{}));
```

Comment:
This isn't wrong, but isn't this more simply `new String[]{"scm1"}`?

Reply:
Yes u r right - will update it